src/components/schemas/report.yaml

```diff
@@ -339,6 +339,20 @@ admin_report_out:
       example: Sent Back
       description: readable state name
 
+approver_report_in:
+  type: object
+  additionalProperties: false
+  required:
+   -purpose
+   -id
+  properties:
+    id:
+      $ref: './fields.yaml#/id_string'
+    purpose:
+      type: string
+      description: |
+        Indicates purpose of the reports. It can only be updated for an existing report before it gets in `APPROVER_PENDING` state.
```

Comment:
`....before it gets in `APPROVER_PENDING` state.` can be changed to `..before the report is moved to `APPROVER_PENDING` state.`

Reply:
This is not fixed yet, Did you push changes yet?